CIScripts/Test/Tests/Pkt0.Tests.ps1

```diff
@@ -0,0 +1,81 @@
+Param (
+    [Parameter(Mandatory=$true)] [string] $TestbedAddr
+)
+
+. $PSScriptRoot\..\Utils\CommonTestCode.ps1
+. $PSScriptRoot\..\TestConfigurationUtils.ps1
+. $PSScriptRoot\..\..\Common\Aliases.ps1
+. $PSScriptRoot\..\..\Common\VMUtils.ps1
+
+. $PSScriptRoot\..\GetTestConfigurationCodiLegacy.ps1
+$TestConf = Get-TestConfiguration
+$Session = New-PSSession -ComputerName $TestbedAddr -Credential (Get-VMCreds)
+
+Describe "vRouter Agent" {
+    Context "given vRouter Forwarding Extension is NOT running" {
+        It "crashes" {
+            Assert-AgentProcessCrashed -Session $Session -After $TimeBeforeCrash | Should Throw
```

Comment:
1. Actually, it should *not* throw
2. Left hand side of `|` should be wrapped in `{}`
3. As the syntax `{ Assert ... } | Should Not Throw` is easy to get wrong (as this example shows), I'd go with a plain `Assert` _for now_